frb_rust/Cargo.toml

```diff
@@ -20,6 +20,8 @@ uuid = { workspace = true, optional = true }
 
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
 allo-isolate = { version = "0.1.14-beta.3", features = ["anyhow"] }
+dart-sys = "2.0"
```

Comment:
hmm if we are using dart-sys, is it possible we do not manually copying those .h files

Reply:
dart_sys creates bindigs to `dart_api.h` and we need function `Dart_InitializeApiDL`  from `dart_api_dl.h`.